md-render/HHVM/UserDocumentation/IncludeExamplesFilter.rb

```diff
@@ -51,7 +51,21 @@ def call
             File.read(full_path),
             language: 'Hack',
           );
-          node.replace(code)
+          code_node = node.replace(code)
+          # If the example has HHVM output associated with it, then print it
+          # after the code example
+          if File.exists? (full_path + ".example.hhvm.out")
```

Comment:
Right now, I always expect a `.example.hhvm.out` for output.

I instruct example creators to directly copy the `.expect` file as an `.example.hhvm.out`; 

if there is an `.expectf`, then I should write that you run the the test with `no-clean` and copy the resulting `.out` file into `.example.hhvm.out`.

We could check for `.hhvm.expect` to optionally avoid the copy. Check for `.example.hhvm.out`, if doesn't exist, then check for `.hhvm.expect`.